### PR TITLE
Addressed mission manager's recurring constants review

### PR DESCRIPTION
**Pull request by tanishqaggarwal**

### CODE REVIEW

File: src/fsw/FCCode/MissionManager.hpp [hunk 1]
```diff
@@ -17,10 +17,10 @@ class MissionManager : public TimedControlTask<void> {
     void execute() override;
 
     // Constants that drive state transitions.
-    static constexpr double detumble_safety_factor = 0.2;
+    WritableStateField<double> detumble_safety_factor_f;
+    WritableStateField<double> close_approach_trigger_dist_f;
```
Comment (nhz2) [inline src/fsw/FCCode/MissionManager.hpp]: Add units.

File: src/fsw/FCCode/MissionManager.cpp [hunk 2]
```diff
@@ -5,13 +5,22 @@
 
 MissionManager::MissionManager(StateFieldRegistry& registry, unsigned int offset) :
     TimedControlTask<void>(registry, "mission_ct", offset),
+    detumble_safety_factor_f("detumble_safety_factor", Serializer<double>(0, 1, 10)),
+    close_approach_trigger_dist_f("trigger_dist.close_approach", Serializer<double>(0, 10000, 14)),
+    docking_trigger_dist_f("trigger_dist.docking", Serializer<double>(0, 100, 10)),
```
Comment (kkrol27) [inline src/fsw/FCCode/MissionManager.cpp]: Do we have plans to change these to some value related to `|∆r| + |∆v|`?

Comment (tanishqaggarwal) [reply src/fsw/FCCode/MissionManager.cpp]: It's tracked on an issue, hopefully we'll get around to it. let's keep things simple for now though?

Comment (tanishqaggarwal) [reply src/fsw/FCCode/MissionManager.cpp]: #150

Comment (kkrol27) [reply src/fsw/FCCode/MissionManager.cpp]: Oh sweet sounds good. And yeah I agree we keep it simple for now.

**Pull request merged**